### DISCUSSION TITLE
Console errors on interaction on the demo page

### DISCUSSION THREAD

**Discussion by Ryuno-Ki** [JulianPrieber/littlelink-custom#199] [Issues]
After creating a demo page, clicking the share link results in an error in the browser console:

`toastBox is not defined`.

Also, it appears to see jQuery (`$ is not defined`) present, which is not the case.

**Comment by JulianPrieber** [comment]
The toast box is something that got removed previously. As far as I can tell, this issue is fixed with new releases.
In case that it isn't I'll make sure this gets fixed.

> Also, it appears to see jQuery ($ is not defined) present, which is not the case.

This one I'm not sure about. It should be something in here: https://github.com/JulianPrieber/littlelink-custom/blob/main/littlelink/js/share.button.js

**Comment by Ryuno-Ki** [comment]
https://github.com/JulianPrieber/littlelink-custom/blob/3c4df6721c3faec1d20a0e2c4d946f12c8925663/littlelink/js/share.button.js#L65

Could be rewritten as:

```js
document.querySelector('#share-share-button').addEventListener('click', () => {
  if (navigator.share) {
    navigator.share({
        title: '',
        text: '',
        url: '',
      })
      .then(() => console.log('Successful share'))
      .catch((error) => console.log('Error sharing', error));
  } else {
    console.log('Share not supported on this browser, do it the old way.');
  }
});
```

See also https://youmightnotneedjquery.com/#on

**Comment by JulianPrieber** [comment]
https://github.com/JulianPrieber/littlelink-custom/commit/604a0f631d880414c0924f3bf6bbdf950640d2c2

**Comment by JulianPrieber** [comment]
This problem only appears on Firefox. The share button opens the system default share dialog on Chrome, but Firefox does not support that.

![chrome_d9qRIKykDr](https://user-images.githubusercontent.com/60265788/193837474-eb8e2f3b-0da2-4c50-87fb-54e8b9a3955f.png)
https://caniuse.com/?search=Navigator%20share
